phpstan.neon.dist

```diff
@@ -0,0 +1,47 @@
+parameters:
+	paths:
+		#- api
+		- Civi
+		#- CRM
```

Comment:
I guess the `api` and `CRM` directories are meant to be ignored temporarily only until we catch up with refactoring?

Reply:
Exactly. I didn't want to change all the files in this PR.